packages/magento-review/components/ProductReviews/ProductReviews.tsx

```diff
@@ -74,21 +75,21 @@ export function ProductReviews(props: ProductReviewsProps) {
         marginTop: theme.spacings.sm,
       })}
     >
-      <PageLink href={`/account/reviews/add?url_key=${url_key}`} passHref>
-        <Button
-          variant='pill'
-          color='primary'
-          size='medium'
-          className={classes.writeReviewButton}
-          sx={(theme) => ({
-            [theme.breakpoints.down('sm')]: {
-              whiteSpace: 'nowrap',
-            },
-          })}
-        >
-          <Trans>Write a review</Trans>
-        </Button>
-      </PageLink>
+      <Button
+        LinkComponent='button'
```

Comment:
```suggestion
```

it is of type=button by default when href isn't passed (which isn't in this case)